engine/language-server/src/main/scala/org/enso/languageserver/libraries/LocalLibraryManager.scala

```
@@ -190,6 +192,21 @@ class LocalLibraryManager(
     _ = saveManifest(manifestPath, updatedManifest)
   } yield EmptyResponse()
 
+  /** Loads the package config for a local library. */
+  private def getPackage(libraryName: LibraryName): Try[GetPackageResponse] =
+    for {
+      libraryRootPath <- localLibraryProvider
+        .findLibrary(libraryName)
+        .toRight(LocalLibraryNotFoundError(libraryName))
+        .toTry
+      configPath = libraryRootPath / Package.configFileName
+      config <- loadPackageConfig(configPath)
+    } yield GetPackageResponse(
+      license         = config.license,
+      componentGroups = config.componentGroups.toOption,
```

It will be an option if it failed to parse, right? (If it parsed correctly it will be an empty list instead, right?)

I don't like that we are swallowing errors. I'd suggest at least logging a warning here, specifying the parse error.
But ideally - the error should be forwarded to the IDE as part of the response payload (i.e. for this specific library instead of having a missing field, there could be some other field indicating that this failed to parse; the bulk operation that parses all libraries could include a list of modules which failed to parse, ideally each with the associated error).

I understand that this adds work and is not crucial for an MVP, so maybe it does not need to be done as part of this task - but I would at least suggest to create a task to deal with it at some point. I think it's pretty important to _not_ swallow errors, as this will become a source of confusion for the users at some point: imagine the scenario where a user edits the component group in such a way that it fails to parse - they will stop seeing the respective component groups but will otherwise get no indication of error.